.github/workflows/local-build.yml

```diff
@@ -31,13 +32,22 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
-      - name: Install Node.js
+      - name: Install Node.js 16
         uses: actions/setup-node@v3
+        if: contains(fromJson('["aws-environment-param-sync"]'), matrix.package) == false
         with:
           node-version: 16.x
           cache: 'npm'
           cache-dependency-path: ${{ matrix.package }}/package-lock.json
 
+      - name: Install Node.js 20
```

Comment:
Have we decided 20 is mature enough now and should be updating https://cartoncloud.atlassian.net/wiki/spaces/PD/pages/60948488/Technology+Stack#Product-Technologies ?

Reply:
I wasn't intending to use Node 20, I originally started with 16. I was forced to because one of the libraries I wanted to use required it. This is probably a good place to first roll out node 20 as it's internal functionality only.

@aarontwf @blake-cartoncloud @tomaszCarton Concerns with using node 20? I'm not familiar enough to comment on whether it is mature enough.

Reply:
Node 20 is what GitHub is using as it's current standard for actions. See https://github.com/actions/checkout?tab=readme-ov-file#whats-new

Node 18 [is the latest LTS version](https://nodejs.org/en), which makes sense to be our standard for app runtimes. This is just actions, not part of the app, so not too big of an issue.